lib-audio/src/main/java/org/rfcx/guardian/audio/flac/FLACStreamEncoder.java

```diff
@@ -0,0 +1,40 @@
+package org.rfcx.guardian.audio.flac;
+
+import java.io.File;
+import java.nio.ByteBuffer;
+
+public class FLACStreamEncoder {
+
```

Comment:
Do a method overload here to also support `encode(outputFile, sampleRate, channels, bitsPerSample, inputFile)`